code/controllers/subsystem/afk.dm

```diff
@@ -21,7 +21,8 @@ SUBSYSTEM_DEF(afk)
 
 /datum/controller/subsystem/afk/fire()
 	var/list/toRemove = list()
-	for(var/mob/living/carbon/human/H in GLOB.alive_mob_list)
+	for(var/thing in GLOB.human_list)
+		var/mob/living/carbon/human/H = thing
```

Comment:
This is going to trigger on dead mobs now; functionality change; not sure if we should do this.

Reply:
It should remove them from the watch list if they are in there due to line 33. `H.stat || ...`. Or they will just be ignored if they are not in the watch list